src/models/tiles/geometry/geometry-content.ts

```diff
@@ -618,21 +601,6 @@ export const GeometryContentModel = GeometryBaseContentModel
       return applyAndLogChange(board, change);
     }
 
-    // TODO Remove this or change metadata to tilecontainer or something
-    function updateTitle(board: JXG.Board | undefined, title: string) {
-      const change: JXGChange = {
-              operation: "update",
-              target: "metadata",
-              properties: { title }
-            };
-      return applyAndLogChange(board, change);
```

Comment:
The logging here is now done by the generic `TileModel#setTitle`, this means the title update event will look a little different but at least it will still be recorded.